util/start_test

```diff
@@ -615,7 +616,7 @@ def check_environment_with_args():
     global logs_dir
     logs_dir = os.path.join(test_dir, "Logs")
     if not os.path.isdir(logs_dir):
```

Comment:
we could probably remove all the "if dir does not exist" checks now, but I don't mind leaving them